packages/expo-module-scripts/tsconfig.node.json

```diff
@@ -1,10 +1,13 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
-
   "compilerOptions": {
+    "target": "ES2022",
+    "module": "Node16",
+    "moduleResolution": "Node16",
```

Comment:
nit: this overlaps with `@tsconfig/node18`. I think if we're basing this off of `node18`, and we don't have a secondary reason to override it, removing this will mean that when we're changing this in the future, we won't assume we had a good reason for particular settings (e.g. indicating that it's afe to have the `target` change here, for example)
https://github.com/tsconfig/bases/blob/056f30ab7bdc641dc1d427cd7d3fdbe17df09d8a/bases/node18.json#L8-L15

Reply:
Yes, it's an another `ncc` related issue as stated in description. It cannot resolve the `extends` properly, when package name is used, not direct path. I was surprised by that.

Reply:
as per DMs, since `extends` doesn't work and this is only for packages that require `ncc` to build, my opinion is that it's non-blocking, and a separate problem we can get to later. Either way, happy to skip, same for the other comment